tools/samtools.wdl

```diff
@@ -267,11 +325,38 @@ task subsample {
 }
 
 task merge {
+    meta {
+        description: "Merges multiple sorted BAMs into a single BAM"
+        outputs: {
+            merged_bam: "The BAM resulting from merging all the input BAMs"
+        }
+    }
+
+    parameter_meta {
+        bams: "An array of BAMs to merge into one combined BAM"
+        prefix: "Prefix for the BAM file. The extension `.bam` will be added."
+        new_header: "Use the lines of FILE as `@` headers to be copied to the merged BAM, replacing any header lines that would otherwise be copied from the first BAM file in the list. (File may actually be in SAM format, though any alignment records it may contain are ignored.)"
+        region: "Merge files in the specified region"
```

Comment:
Is there a format for this? Presumably the same as in other `samtools` commands, something like `chr:start-end`.

Reply:
The docs didn't say. I assume it's the standard Samtools format of `chr:start-end`. We _should_ write here what format is, but as Samtools docs don't mention it we're left with figuring it out through trial and error.

Do you know of any BAMs it would be easy to test with?

Reply:
I don't have any specific BAMs. I would probably just pick two of the same data type (e.g. WES?) and try it out with a region specification. Then verify that only those regions were output. Then we can just do something like `(Format: `chr:start-end`)`. If other formats work, that's fine, but I think as long as we document one that works, that's sufficient.